grid/payments/create-product/profile.supr

```diff
@@ -0,0 +1,76 @@
+name = "payments/create-product"
+version = "1.0.0"
+
+"""
+Create product  usecase
```

Comment:
suggestion (docs): This could use a further description. Why should I create a product? Where can I use it further?